cps/src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/block.jelly

```diff
@@ -50,6 +50,7 @@ THE SOFTWARE.
             </j:forEach>
         </d:tag>
     </d:taglib>
+    <f:entry title="Workflow DSL Reference" help="/${it.DSL_REF_URL}"/>
```

Comment:
I suppose this is intended to supersede the block at lines 98–100, which you forgot to delete (and which is still using the deleted `STATIC_URL`).